about.html

```diff
@@ -0,0 +1,144 @@
+      <div class="col-md-8  ms-5 text-center">
+        <h2 class="h1-text header-para">WOMEN IN TECH<br>
+        GLOBAL CONFERENCE
+    2022</h3>
+     <div class="card mb-5" style=" font-size:12px">
+  <div class="card-body bg-white">
+    
+    <p class="about-section">The women in tech event is  conference that brings together women passionate about technology from all over the world. These women will get a chance to showcase their work to fellow women tech enthusiasts. </p>
+    </p>
+  </div>
+</div>
+
+<div class="about-email">
+    <P>Please contact us via email for further questions regarding the women in tech global conference 2022<br>
+    <u><b>@womenintechconference@hotmail.com</b></u></P>
+      </div>
+
+      
+     
+    </section>
+
+       <section class="logo-section" id="logo-section">
+      <div class="about-section">
+        <h2 class="logo-heading">Women In Tech Logo</h2>
+        <hr />
+        <p class="about-para">
+          The logo of the women in tech brand is meant to represent diversity and allow any woman to embrace technology regardless of their race or nationality.
+        </p>
+        <img
+          src="./assets/images/logo.png"
+          alt="Women in Tech logo"
+          class="about-logo"
+        />
+      </div>
+    </section>
+
+ 
+
+     <section class="past-conferences" id="past-conferences">
+      <div class="about-section">
+        <h2 class="logo-section">See the past Women in Tech Conferences</h2>
+        <hr />
+        <p>
+          Take a look at the last twoWomen in Tech conferences from 2020 to 2021.
+        </p>
+        <div class="images">
+          <div class="fashion-images">
+            <img
+              src="./assets/images/conference1.jpg"
+              alt="People in an event"
+              class="fashion-img"
+            />
+            <div class="blend">
+              <h2>2021</h2>
+              <p>Women in Tech 2021 in South Africa.</p>
+            </div>
+          </div>
+          <div class="fashion-images">
+            <img
+              src="./assets/images/conference2.jpg"
+              alt="People checking items in a fashion event"
+              class="fashion-img"
+            />
+            <div class="blend">
+              <h2>2021</h2>
+              <p>Women in Tech 2020 in Kenya.</p>
+            </div>
+          </div>
+        </div>
+      </div>
+    </section>
+
+  <!-- Footer Section -->
+    <footer>
+      <div class="footer-section">
+        <div class="footer-descr">
+          <img src="./assets/images/logo.png" alt="The cc logo" class="copyright-img" />
+        </div>
+        <div>
+          <span>2022 Women in Tech Global Conference</span>.Some Rights Reserved</span> <br />
+          <span> 在这里度过了一段美好的时光，而不是很长时间</span>
+        </div>
+      </div>
+    </footer>
+    <script src="./scripts/index.js"></script>
+    <script src="./scripts/mobilemenu.js"></script>
+  </body>
+</html>
```

Comment:
While fixing some of the issues raised by previous reviewers, your layout for the about page is kinda wacky. There is some horizontal scroll on this page and also the navbar in the mobile version is being cut off on the left side. Consider fixing these two issues.
Here is a snapshot for your reference.
![image](https://user-images.githubusercontent.com/33268927/175472846-3d60bd47-a3bc-4a98-84b5-058eee2dcef9.png)